src/screen.rs

```diff
@@ -67,6 +71,8 @@ impl Interfaceable for PrettyPrompt {
             }
             Err(_) => String::from("quit"),
         };
+        self.editor.save_history(".history.txt").unwrap();
```

Comment:
This should only run if history is true

Reply:
Also you should unwrap safely

Reply:
Word, I was thinking I might need to put a check there.